pyfar/coordinates.py

```diff
@@ -460,39 +423,31 @@ def get_sph(self, convention='top_colat', unit='rad', convert=False):
     def set_cyl(self, points_1, points_2, points_3,
                 convention='top', unit='rad'):
```

Comment:
For cylindrical coordinates, the only possible convention parameter is `'top'`, isn't it?

Reply:
see above - keeping it is more flexible for future extensions